src/main/java/com/target/ready/library/system/Entity/Book.java

```diff
@@ -0,0 +1,73 @@
+package com.target.ready.library.system.Entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+
+public class Book {
```

Comment:
use PropertyNamingStrategy.SNAKE_CASE to make sure all the request/response attributes are in snake case.